src/engraving/libmscore/note.cpp

```diff
@@ -2766,7 +2766,10 @@ void Note::updateRelLine(int relLine, bool undoable)
 
     const Staff* staff  = score()->staff(idx);
     const StaffType* st = staff->staffTypeForElement(this);
-
+    if (st->isTabStaff()) {
+        // tab staff is already correct, and the following relStep method doesn't apply whatsoever to tab staves
+        return;
+    }
```

Comment:
3.x additionally has this it that place:
´``cpp
      if (chord()->staffMove()) {
            // check that destination staff makes sense (might have been deleted)
            int minStaff = part()->startTrack() / VOICES;
            int maxStaff = part()->endTrack() / VOICES;
            const Staff* stf = this->staff();
            if (idx < minStaff || idx >= maxStaff || st->group() != stf->staffTypeForElement(this)->group()) {
                  qDebug("staffMove out of scope %d + %d min %d max %d",
                     staffIdx(), chord()->staffMove(), minStaff, maxStaff);
                  chord()->undoChangeProperty(Pid::STAFF_MOVE, 0);
                  }
            }

```
Not needed on master?